dub.selections.json

```diff
@@ -3,16 +3,17 @@
 	"versions": {
 		"botan": "1.12.9",
 		"botan-math": "1.0.3",
-		"diet-ng": "1.4.3",
+		"diet-ng": "1.4.4",
 		"eventcore": "0.8.27",
 		"hyphenate": "1.1.1",
 		"libasync": "0.8.3",
 		"libdparse": "0.7.1",
 		"libevent": "2.0.2+2.0.16",
 		"memutils": "0.4.9",
 		"openssl": "1.1.6+1.0.1g",
-		"taggedalgebraic": "0.10.8",
-		"vibe-core": "1.3.0",
-		"vibe-d": "0.8.2"
+		"stdx-allocator": "2.77.0",
+		"taggedalgebraic": "0.10.9",
+		"vibe-core": "1.4.0-alpha.1",
+		"vibe-d": "0.8.3-alpha.1"
```

Comment:
`0.8.3-alpha.2` fails with:

```
../../../../vibe/vibe.d-repo/stream/vibe/stream/base64.d(10,8): Error: module stream is in file 'vibe/core/stream.d' which cannot be read
import path[0] = ../../../../vibe/vibe.d-repo/stream
import path[1] = /usr/include/dl
```

I guess that's an error due to 	`targetType "none"` in `core/dub.sdl`?

Reply:
Hm, not sure about that, I'll test locally. The same error still happens for https://github.com/vibe-d/vibe.d/pull/2033 on DMD 2.072.x because the latest DUB isn't used anymore.

Reply:
The error was a regression in `master` (which is used on Project-Tester), and Martin already submitted a fix: https://github.com/dlang/dub/pull/1350

So I will check whether this is still needed.

> DMD 2.072.x because the latest DUB isn't used anymore.

Yeah we should allow opt-in upgrade to the latest DUB version in the installer script.

Reply:
So the bump is required for another PR too - https://ci.dlang.io/blue/organizations/jenkins/dlang-org%2Fphobos/detail/PR-5921/24/pipeline

```
diet-ng 1.4.3: building configuration "library"...
vibe-d:utils 0.8.2: building configuration "library"...
../../../../../.dub/packages/vibe-d-0.8.2/vibe-d/utils/vibe/internal/memory_legacy.d(187,37): Error: constructor `vibe.utils.hashmap.HashMap!(void*, ulong, DefaultHashMapTraits!(void*)).HashMap.this(IAllocator allocator)` is not callable using argument types `(RCIAllocator)`
../../../../../.dub/packages/vibe-d-0.8.2/vibe-d/utils/vibe/internal/utilallocator.d(14,85): Error: cannot implicitly convert expression `(*function () => allocatorObject(instance))()` of type `RCIAllocator` to `std.experimental.allocator.IAllocator`
dmd failed with exit code 1.
[ERROR] /var/lib/jenkins/dlang_projects@2/dlang/dub/test/ddox.sh:11 command failed
[ERROR] /var/lib/jenkins/dlang_projects@2/dlang/dub/test/ddox.sh:11 command failed
```

https://github.com/dlang/phobos/pull/5921